app/mailers/card_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class CardMailer < ActionMailer::Base
+  default from: "sichirc@gmail.com"
+
+  def pending_cards_notification(user, cards_for_review_count)
+    @user = user
+    @cards_for_review_count = cards_for_review_count
+    @url  = 'http://127.0.0.1:3000'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.